cmd/sidecars/network-passt-binding/server/server.go

```diff
@@ -49,13 +50,19 @@ func (s InfoServer) Info(_ context.Context, _ *hooksInfo.InfoParams) (*hooksInfo
 				Name:     hooksInfo.OnDefineDomainHookPointName,
 				Priority: 0,
 			},
+			{
+				Name:     hooksInfo.ShutdownHookPointName,
+				Priority: 0,
+			},
 		},
 	}, nil
 }
 
-type V1alpha2Server struct{}
+type V1alpha3Server struct {
+	Done chan struct{}
```

Comment:
I'm missing how this was initialized.

Reply:
It is part of the `main` method.
`hooksV1alpha3.RegisterCallbacksServer(server, srv.V1alpha3Server{Done: shutdownChan})`